R/parameters.R

```diff
@@ -164,7 +164,9 @@ unk_check <- function(x) {
 print.parameters <- function(x, ...) {
   x <- tibble::as_tibble(x)
 
-  cat("Collection of", nrow(x), "parameters for tuning\n\n")
+  cli::cli_par()
+  cli::cli_text("Collection of {nrow(x)} parameters for tuning")
+  cli::cli_end()
```

Comment:
`cli::cli_par()` starts a paragraph and `cli::cli_end()` ends it. This way we get neat spacing between the blocks of output without having to use `\n`